### ISSUE TITLE
Background processing after process_request

### ISSUE THREAD

**Issue by kmeekva** [jaxon-php/jaxon-core#139]
I have a function being called with    $jaxon->processRequest();  that is called from a client browser.

The PHP function uses register_shutdown_function() php command to queue up another function that takes a long time to run which I want to execute in the background AFTER the object response has been returned to the client.

In the past I've forced the flushing of output to the client using these commmands after the processRequest.

ob_flush();
ob_end_flush();
session_write_close();
fastcgi_finish_request();
die(); //then the function registered will run on server after 

But I notice with version 4 these commands never get exectued if they come after $jaxon->processRequest();   In earlier versions they did get executed and it worked fine.

I tried putting them in the after callback -- but when that is called the object response to the client has not been sent yet so I can't flush the output and exit?

Is there a way to reference the response to be sent in the callback so I can just print it and do the flushing to "complete" the php program so data is sent and the client connection is closed BEFORE the Long running shutdown function is executed?

As it stands since I can't flush and close the  client session -- the response doesn't get flushed until after my long background processes completes.

The problem is that I am trying to queue up a slow function to run in a PHP relong running function (sending mail)

**Comment by kmeekva** [comment]
I found the option   **core.process.exit**

$jaxon->setOption('core.process.exit', true);

Which causes the exit() to be called when the processRequest is finished.  

However I have found that this is not enough to "FORCE" the connection to the client to be totally flushed and closed so the client will not have to wait for the register_shutdown function to complete.

The solution I use is to do:

ob_flush();
ob_end_flush();
session_write_close();
fastcgi_finish_request();

Before the exit. 

I will submit a pull request to add these commands to the AjaxSendTrait.php file.
